learn/wavemaker-release-notes/v10-7-3.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "WaveMaker 10.7.3 - Release date: 12 July 2021"
+id: ""
+sidebar_label: "Release v10.7.3"
+---
+*WaveMaker releases an update with bug fixes.*
+
+## Bug Fixes
+
+---
+
+- Fixed an issue with GoogleMaps prefab showing only the *RoadMap* view in preview, even when selecting the Map View Type as Satellite, Terrain, or Hybrid.
+- Fixed an issue related to an application deployed in root context with OpenID authentication.
+- Fixed an issue related to the DateTime widget not opening appropriately when modifying via date picker.
+- Fixed an issue related to the Chips widget with min char value set to `0` in Live Form field, for example, that loads a widget dropdown by default.
+
+### Form
+
+- Fixed an issue related to the Change event not triggering when using Delete or Backspace keys.
```

Comment:
Fixed an issue related to the Change event not triggering when the whole value in the input box is selected and delete/backspace key is pressed.